app/assets/stylesheets/_layout.scss

```diff
@@ -0,0 +1,19 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+figure {
```

Comment:
`figcaption` provides a lot more flexibility. For example, you can have multiple images within a `figure` in which one `figcaption` can describe. Also, `figcaption` can include other HTML elements, which `alt` is limited in (think rich, visual text describing something).